index.js

```diff
@@ -188,12 +192,12 @@ export default class OTPInputView extends Component {
                         this.handleChangeText(index, text)
                     }}
                     onKeyPress={({ nativeEvent: { key } }) => { this.handleKeyPressTextInput(index, key) }}
-                    value={digits[index]}
-                    keyboardType={keyboardType}
-                    textContentType={isAutoFillSupported ? "oneTimeCode" : "none"}
+                    value={!clearInputs ? digits[index]: ""}
+                    keyboardType="number-pad"
+                    textContentType= {isAutoFillSupported ? "oneTimeCode" : "none"}
+
                     key={index}
                     selectionColor="#00000000"
-                    secureTextEntry={secureTextEntry}
```

Comment:
@hasithaWalpola why are we deleting this?

Reply:
It was deleted mistakenly. It should be there. Sorry for that